MaxMind.MinFraud/Response/MultiplierReason.cs

```diff
@@ -0,0 +1,200 @@
+﻿using System.Text.Json.Serialization;
+
+namespace MaxMind.MinFraud.Response
+{
+    /// <summary>
+    /// The risk score reason for the multiplier.
+    /// </summary>
+    /// <remarks>
+    /// This class provides both a machine-readable code and a human-readable
+    /// explanation of the reason for the risk score, see
+    /// https://dev.maxmind.com/minfraud/api-documentation/responses/#schema--response--risk-score-reason--multiplier-reason.
+    /// </remarks>
+    public sealed class MultiplierReason
```

Comment:
I had to rename it to `MultiplierReason` due to `'Reason': member names cannot be the same as their enclosing type` error, see https://github.com/maxmind/minfraud-api-dotnet/actions/runs/10639043875/job/29496249067?pr=259.